src/main/java/org/terasology/blockNetwork/BlockNetworkUtil.java

```diff
@@ -40,6 +53,13 @@ private static Rotation getRotation(Block block) {
         return rotation;
     }
 
+    /**
+    * Give the new side of the block based on its' rotation
+    *
+    * @param block - the block that is about to be checked its' rotation and counted its' connections
+    * @param definedSides - the definedSides on which the loop iterates the sides of the block
+    * @return a byte of sides
+    */
     public static byte getResultConnections(Block block, byte definedSides) {
```

Comment:
Overall having read this javadoc I'm still confused as to what this method actually does, the description talks about getting new sides but the method name talks about getting connections. Is the method returning sides that it's connected on?